features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/developer/DeveloperSettingsPresenter.kt

```diff
@@ -90,8 +90,8 @@ class DeveloperSettingsPresenter(
         }
 
         LaunchedEffect(Unit) {
-            FeatureFlags.entries
-                .filter { it.isFinished.not() }
+            featureFlagService.getAvailableFeatures()
```

Comment:
`getAvailableFeatures()` is already filtering with `!isFinished` so the filter below could be simplified.
Or maybe to reduce duplication of code, `getAvailableFeatures` could take a boolean parameter `isInLabs` to handle all the filtering?

Reply:
It's actually duplicated so we can check this has the right behaviour in unit tests (i.e. if `getAvailableFeatures` returned some unfinished features we'd still remove them).

> Or maybe to reduce duplication of code, `getAvailableFeatures` could take a boolean parameter isInLabs to handle all the filtering?

I thought about that, but this seems more related to how we want to handle some data in the UI than something that should be part of the logic of the service.